Makefile

```diff
@@ -337,6 +339,21 @@ crc_storage_cleanup: ## cleanup local storage PVs in CRC vm
 	if oc get sc ${STORAGE_CLASS}; then oc delete sc ${STORAGE_CLASS}; fi
 	bash scripts/delete-pv.sh
 
+##@ OPERATOR_NAMESPACE
+.PHONY: operator_namespace
+operator_namespace: export NAMESPACE=${OPERATOR_NAMESPACE}
+operator_namespace: ## creates the namespace specified via OPERATOR_NAMESPACE env var (defaults to openstack)
```

Comment:
The comment here should read  `defaults to openstack-operators`, shouldn't it?

Reply:
yes, good catch